pages/resources/organizations/rse-groups.md

```diff
@@ -1,10 +1,12 @@
 ---
 layout: page
-title: Organizations with RSEs
-subtitle: Organizations around the United States that employ groups of RSEs
+title: Institutional RSE Groups in the United States
 menubar: organizations
 permalink: /resources/organizations/rse-groups/
 ---
+
+This is an incomplete list of RSE groups at US institutions that take on a range of projects and are open to collaborations with researchers within the scope of their support.  RSEs work at many places beyond these groups.  
```

Comment:
The "with" in the title is kind of intentional - we list MIT and NIH and such that don't have RSE groups (that we know of), we already list places that have RSE's beyond RSE groups, so long as they still have plenty scattered about.

Reply:
We've fixed the groups that don't have names - I contacted the people who added those - MIT we have a name, NIH is OK to be removed; Alabama is pending (removed for now - can add back).  We realize this is a slight refocus of this page.  It's intentional because otherwise it doesn't have a clear purpose.  If we need/want another page that lists groups "with" RSEs, we can add one.